inheritance/lib/src/test/java/inheritance/RestaurantTest.java

```diff
@@ -0,0 +1,24 @@
+package inheritance;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class RestaurantTest {
+    @Test
+    public void testRestaurant(){
```

Comment:
You should probably split this into two separate tests, one that tests restaurants and one that test reviews. Tests look good however!